OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -129,7 +129,7 @@ public class ScriptContext : IDisposable
 		public World World { get; private set; }
 		public WorldRenderer WorldRenderer { get; private set; }
 
-		protected readonly MemoryConstrainedLuaRuntime Runtime;
+		public readonly MemoryConstrainedLuaRuntime Runtime;
```

Comment:
Can you please make this `internal` instead?  I'd like to avoid leaking this into mod code.